### PR TITLE
feat : basic html and css setup of e-commerce

### PR DESCRIPTION
**Pull request by ravikumar1002**
E-commerce folder setup
   Changed old e-commerce screen into react

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,9 +1,28 @@
...
+import { Routes, Route } from "react-router-dom";
+import Header from "./components/header/Header";
+import Home from "./pages/home/home";
+import ProductsPage from "./pages/products/products-combine-page";
+import WishlistPage from "./pages/wishlist/wishlsit";
+import LoginPage from "./pages/login/login";
+import ProfilePage from "./pages/profile/profile";
+import SignUpPage from "./pages/signup/signup";
+import CartPage from "./pages/cart/cart";
+import Mockman from "mockman-js";
```
Comment (romabulani) [inline src/App.js:11]: Suggestion : You can read about barrel exports and absolute paths, it will make the export paths shorter

**Review by romabulani — APPROVED** [review]
- class names are self explanatory
- good use of semantic tags
- good to merge

**Review by shrey27 — APPROVED** [review]
File structure looks good
PR is good to merge.

File: src/App.js (hunk 1 above)
Comment (ravikumar1002) [reply src/App.js:11]: yeah sure, next time I'll keep that in mind

**Pull request merged**